db/db.go

```diff
@@ -31,5 +36,68 @@ func ConnectDB(URL string) *mongo.Client {
 	}
 
 	fmt.Println("Connected to MongoDB!")
-	return client
+	return &DBClient{
+    URL: url,
+    client: client,
+  }
+}
+
+//Insert a user in db
+func (c *DBClient)InsertUser(usr *models.User) error{
+  collection := c.client.Database("production").Collection("users")
+  insertResult, err := collection.InsertOne(context.TODO(), usr)
+  if err != nil {
+      log.Fatal(err)
```

Comment:
if it is log.Fatal, the code will panic, instead you should `return err`
change this in the remaining functions too